react/AppIcon/Readme.md

```diff
@@ -1,6 +1,24 @@
-`<AppIcon />` uses a `fetchIcon` prop to load asynchronously an app icon.
+`<AppIcon />` renders an app icon.
 
-## Example with a fake instance of cozyClient
+It contains business logic related to apps documents.
+
+Apps documents provided by the stack are containing a `links.icon` property which is public and can be directly used as `src` value for an `<img />` tag.
+
+`<AppIcon />` is also able to retrive an icon for a registry app. If no attribute `links.icon` is present on the app document, `<AppIcon />` tries to fetch the icon with a registry link, computed thanks to the app attributes `slug` and `latest_version.version`.
+
+### Example
+```
+  <div>
+    <p>
+      <AppIcon app={app} />
+    </p>
+  </div>
+```
+
+## Using fetchIcon
+Is it also possible to provide a custom asynchronous `fetchIcon` which takes an app document as argument and resolves with an icon URL (see [`URL.createObjectURL`](https://developer.mozilla.org/en/docs/Web/API/URL/createObjectURL)). `fetchIcon` may also throw errors.
```

Comment:
I would not document this if it is deprecated.

Reply:
Actually I not sure anymore we should deprecate it.